src/app/indexedDb/atmosphere-db.service.ts

```diff
@@ -26,7 +26,7 @@ export class AtmosphereDbService {
     return this.dbService.delete(this.storeName, materialId);
   }
 
-  updateWithObservable(material: AtmosphereSpecificHeat): Observable<Array<AtmosphereSpecificHeat>> {
+  updateWithObservable(material: AtmosphereSpecificHeat): Observable<any> {
```

Comment:
I think the returned value from "update" calls is the updated object. The return type would be the same as the object, in this case: AtmosphereSpecificHeat